### DISCUSSION TITLE
I'm looking for a process to split multipage tiffs in a specific format

### DISCUSSION THREAD

**Discussion by valdier1** [ImageMagick/ImageMagick#5300] [Help] [answered]
I'm looking to split a file, from a multipage tiff into unique images, renaming the extension numerically.

example: ABC.TIF (5 page)

would become:
ABC.001
ABC.002
ABC.003
ABC.004
ABC.005

I know there is a syntax for managing this, I just can't quite seem to figure it out properly, suggestions?

**Comment by valdier1** [comment, marked as answer]
I resolved this myself sorry:

magick *.tif -set filename:f "%[t]" -scene 1 +adjoin "outputfolder\\%[filename:f].%03d"

**Reply by dlemstra** [reply]
Thanks for coming back and posting the answer.

**Reply by valdier1** [reply]
Very welcome! The number of times I have been helped here, the last I can do is post back something for someone else in the future :)
